### ISSUE TITLE
exception when using Argon2PasswordHasher first in PASSWORD_HASHERS

### ISSUE THREAD

**Issue by nicokant** [merixstudio/django-trench#74]
Hi,
I use cookiecutter-django in a project, which by default use in this settings
```
PASSWORD_HASHERS = [
    # https://docs.djangoproject.com/en/dev/topics/auth/passwords/#using-argon2-with-django
   "django.contrib.auth.hashers.Argon2PasswordHasher",
   "django.contrib.auth.hashers.PBKDF2PasswordHasher",
   "django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher",
   "django.contrib.auth.hashers.BCryptSHA256PasswordHasher",
]
```
This seems to create a problem with login code verification, in particular when using Argon2PasswordHasher as first hasher

**Comment by mostaszewski** [comment]
Hello @nicokant, could You please post a little bit more information about this issue? At least information about raised exception. We will do our best to help You :-)

**Comment by wmaciejewskimer** [comment]
Fixed in the 0.3.0 release https://github.com/merixstudio/django-trench/pull/82#issuecomment-981698082
